app/helpers/format-float.js

```diff
@@ -0,0 +1,7 @@
+import { helper } from '@ember/component/helper';
```

Comment:
Waarschijnlijk kunnen we hieruit https://ember-intl.github.io/ember-intl/versions/master/docs/helpers/format-number#format-number-options wel éen en ander leren als we dit vaken moeten doen ...

Reply:
ik vond niet direct iets over bestaande helpers, daarom heb ik er snel zelf eentje gemaakt.

Reply:
heb nu format-number gebruikt.
echter moeten gaan voor style decimal met maximum digits 0
Want de style percent gaf 100% aan als 10.000 %